protos/dcs/mission/v0/mission.proto

```diff
@@ -443,6 +443,14 @@ message StreamEventsResponse {
     google.protobuf.Struct details = 2;
   }
 
+  /**
+   * Fired every second containing the FPS since the previous event.
+   */
+  message FpsEvent {
+    // The FPS since the last FpsEvent.
+    double fps = 1;
```

Comment:
Some additional ideas/thoughts:
1. Should we rename this to `average`?
2. Should we add a `min` and `max` as well? (for me, DCS easily peaks at over 100FPS, but the average for the second still ends up being at about 70 FPS)

Reply:
> I am assuming you mean the max and min is based on the quickest / slowest time between two individual frames in the second?
Yes 👍

Reply:
I have a local branch with the highest/lowest FPS, but didn't include it in the PR yet. I always got something like: 70 average, 60 lowest, and 250 highest. The 250 highest was almost always reported as the highest. It looks like there are always one or a couple of simulation frames that are pretty quick. With that, I am unsure if it is still helpful information. Feel free to let me know if you'd still like it included.

Reply:
given what you have discovered lets go with the average for the moment; that should still be enough to give server owners insight into what is going on.